config/locales/fr.yml

```diff
@@ -368,6 +432,72 @@ fr:
     click_to_show: Cliquer pour afficher
     reblogged: partagé
     sensitive_content: Contenu sensible
+  terms:
+    body_html: |
+    <h2>Politique de confidentialité</h2>
```

Comment:
I think the syntax is

```yaml
key: |
  indented as if it's a child
  etc
```

CI seems to choke on this line. (could not find expected ':' while scanning a simple key at line 437 column 5)